cg/meta/workflow/analysis.py

```diff
@@ -250,9 +252,10 @@ def upload_bundle_statusdb(self, case_id: str, dry_run: bool = False) -> None:
         new_analysis: Case = self.status_db.add_analysis(
```

Comment:
Should this return an Analysis?
```suggestion
        new_analysis: Analysis = self.status_db.add_analysis(
```